test/PizzeriaTest.cpp

```diff
@@ -52,16 +56,47 @@ TEST_F(PizzeriaTest, completeOrderWithStubPizza)
 TEST_F(PizzeriaTest, calculatePriceForPizzaMock)
 {   
     // Given
-    PizzaMock* mock = new PizzaMock{};
-    Pizzas pizzas = {mock};
-    EXPECT_CALL(*mock, getPrice()).WillOnce(Return(40.0));
-    
+    PizzaMock mock{};
+    Pizzas pizzas = {&mock};
+    EXPECT_CALL(mock, getPrice()).WillOnce(Return(40.0));
+    EXPECT_CALL(mock, getName()).WillOnce(Return(""));
+    EXPECT_CALL(mock, getBakingTime()).WillOnce(Return(minutes(0)));
+    EXPECT_CALL(timerMock, sleep_for(minutes(0))).Times(1);
+
     // When
     auto orderId = pizzeria.makeOrder(pizzas);
+    pizzeria.bakePizzas(orderId);
+    pizzeria.completeOrder(orderId);
     auto price = pizzeria.calculatePrice(orderId);
 
     // Then
     ASSERT_EQ(40, price);
 
-    delete mock;
+}
+
+TEST_F(PizzeriaTest, calculatePriceFor3PizzasOrder)
+{   
+    // Give
+    StrictMock<PizzaMock> funghiMock{};
+    NiceMock<PizzaMock> margheritaMock{};
+    Pizzas pizzas = {new PizzaStub{"STUB"},  &funghiMock, &margheritaMock};
+
+    EXPECT_CALL(funghiMock, getPrice()).WillOnce(Return(40.0));
+    EXPECT_CALL(funghiMock, getName()).WillOnce(Return("Funghi"));
+    EXPECT_CALL(funghiMock, getBakingTime()).WillOnce(Return(minutes(5)));
+    EXPECT_CALL(margheritaMock, getPrice()).WillOnce(Return(25.0));
+    EXPECT_CALL(margheritaMock, getName()).WillOnce(Return("Margherita"));
+    EXPECT_CALL(margheritaMock, getBakingTime()).WillOnce(Return(minutes(9))); 
+    EXPECT_CALL(timerMock, sleep_for(minutes(1))).Times(1);
+    EXPECT_CALL(timerMock, sleep_for(minutes(5))).Times(1);
+    EXPECT_CALL(timerMock, sleep_for(minutes(9))).Times(1);
+
+    // When
+    auto orderId = pizzeria.makeOrder(pizzas);
+    auto price = pizzeria.calculatePrice(orderId);
+    pizzeria.bakePizzas(orderId);
+    pizzeria.completeOrder(orderId);
+
+    // Then
+    ASSERT_EQ(75, price);
```

Comment:
you have some magic numbers. Use constexpr for associated values:
```cpp
constexpr double funghiPrice = 40.0;
...
EXPECT_CALL(funghiMock, getPrice()).WillOnce(Return(funghiPrice));
...
ASSERT_EQ(funghiPrice + margheritaPrice, price);
```

The same for baking time